File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/non-plagiarized/02/T04.ts
package IR;

/**
 *
 * @author 65FBEF05E01FAC390CB3FA073FB3E8CF (452BF208BF901322968557227B8F6EFE010)
 */
class T04 {
    public static main(args: string[]): void {
        console.log("Miles       Kilometers");
        for (let i = 1; i <= 10; i++) {
            if (i === 10) {
                console.log(`${i}          ${i * 1609}`);
            } else {
                console.log(`${i}           ${i * 1609}`);
            }
        }
    }
}

File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/non-plagiarized/02/T04.kt
package IR

/**
 *
 * @author 65FBEF05E01FAC390CB3FA073FB3E8CF (452BF208BF901322968557227B8F6EFE010)
 */
fun main() {
    println("Miles       Kilometers")
    for (i in 1..10) {
        if (i == 10) {
            println("$i          ${i * 1609}")
        } else {
            println("$i           ${i * 1609}")
        }
    }
}

